routes/web.php

```diff
@@ -13,14 +13,19 @@
 |
 */
 
-// Route::get('/', function () {
-//     return view('welcome');
-// });
-
 Route::get('/', function () {
+    return redirect()->route('login');     
+});                                    
+
+
+
+Route::get('/dashboard', function () {
     return view('temp');
 });
 
+Auth::routes();
+
```

Comment:
replaced this way with:
use App\Http\Controllers\HomeController;
and using it in route:
Route::get('/home', [HomeController::class, 'index'])->name('home');